source/codegen/service_helpers.py

```diff
@@ -250,10 +250,9 @@ def _create_param(parameter, expand_varargs=True, repeated_parameters=None):
             return f"{type[:-2]}* {name}"
         else:
             return f"{type[:-2]} {name}[{array_size}]"
-    elif common_helpers.is_pointer_parameter(parameter):
-        return f"{type}* {name}"
     else:
-        return f"{type} {name}"
+        pointer_qualifier = "*" * common_helpers.levels_of_pointer_indirection(parameter)
```

Comment:
Note: Changes in this file were taken directly from https://github.com/ni/grpc-device/pull/607, will go away after that goes in and this branch is rebased.